openslides/utils/views.py

```diff
@@ -88,14 +52,33 @@ def as_view(cls, *args, **kwargs):
         return ensure_csrf_cookie(view)
 
 
-class PDFView(PermissionMixin, View):
+class PDFView(View):
     """
     View to generate an PDF.
     """
-
     filename = ugettext_lazy('undefined-filename')
     top_space = 3
     document_title = None
+    required_permission = None
+
+    def check_permission(self, request, *args, **kwargs):
+        """
+        Checks if the user has the required permission.
+        """
+        if self.required_permission is None:
+            return True
+        else:
+            return request.user.has_perm(self.required_permission)
```

Comment:
I like this, but I think, that you don't.

If there is a PDF that only the manager can see (for example the user Password PDF), which permission has to be defined in required_permission? Only 'can_manage' without 'can_see'?

You have either to redefine the can_manage permission to also include 'can_see', or change the required_permission attribute to an list or tuple.

I would like, that we redefine the can_manage permission. You do not have to do this in this pull request. You can leave this method as it is and create an issue to redefine the can_manage permission